exporter.go

```diff
@@ -55,6 +57,8 @@ var (
 	scrapeURI = flag.String("nginx.scrape-uri", defaultScrapeURI,
 		`A URI for scraping NGINX or NGINX Plus metrics.
 	For NGINX, the stub_status page must be available through the URI. For NGINX Plus -- the API. The default value can be overwritten by SCRAPE_URI environment variable.`)
+	tlsSkipVerify = flag.Bool("nginx.tlsSkipVerify", defaultTlsSkipVerify,
```

Comment:
could you possibly rename this flag from `nginx.tlsSkipVerify` to `nginx.ssl-verify` and make its default value `true`?   in this case, if the user wants to disable ssl verification, they will need to specify `-nginx.ssl-verify=false` when running the exporter. Could you also update the doc string, the env variable name as well as variable names accordingly? 

When coming up with the flag names we used the official [haproxy exporter](https://github.com/prometheus/haproxy_exporter) as a guide and would like to stay consistent with their argument naming scheme.

Reply:
Thanks for the comments, I have made the requested changes

Let me know if there is anything else